crates/services/upgradable-executor/wasm-executor/src/ext.rs

```diff
@@ -148,7 +148,10 @@ pub fn input(size: usize) -> anyhow::Result<InputDeserializationType> {
 }
 
 /// Gets the next transactions by using the host function.
-pub fn next_transactions(gas_limit: u64) -> anyhow::Result<Vec<MaybeCheckedTransaction>> {
+pub fn next_transactions(
+    gas_limit: u64,
+    _max_transactions: u16,
```

Comment:
Transactions are fetched via a host function here. I am not sure how to retrieve up to `max_transactions` of them without changing the host function interface (assuming that under the hood the host function implementation will change the host state and remove the transactions)